modules/op-resolver-git-config-anon.adoc

```diff
@@ -0,0 +1,37 @@
+// This module is included in the following assembly:
+//
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+
+:_content-type: PROCEDURE
+[id="resolver-git-config-anon_{context}"]
+= Configuring the Git resolver for anonymous Git cloning
+
+If you want to use anonymous Git cloning, you can optionally configure the default Git revision, fetch timeout, and default repository URL for pulling remote pipelines ans tasks from a Git repository.
+
+.Procedure
+
+. To edit the `TektonConfig` custom resource, enter the following command:
++
+[source,terminal]
+----
+$ oc edit TektonConfig config
+----
++
+. In the `TektonConfig` custom resource, in the `pipeline.git-resolver-config` spec, specify  configuration:
```

Comment:
Seems like you're missing some words here with "specify configuration". Should it be "specify the following configuration" ?  (multiple files)